CONTRIBUTING.md

```diff
@@ -1,167 +1,113 @@
 
- * f. [pre-flight checks](#f-pre-flight-checks)
+4. Don't make multiple unrelated changes in a single commit. Only strongly related changes must be included in a single commit.
+5. Every commit should be meaningful. For example, don't create a new commit to fix a typo you made in your previous commit.
+6. If minified files are not already available in the library you are trying to add, minify CSS and JavaScript files using [UglifyJS](https://www.npmjs.com/package/uglify-js "UglifyJS") or [web-minify-helper](https://github.com/PeterDaveHello/web-minify-helper "web-minify-helper"). Append `.min` to minified files (for example, `jquery.min.js`).
+* Filenames should not consist of a version number and must be lowercase. `useful.min.js` is OKAY, but `useful-2.0.1.min.js` is NOT.
 
- * g. [P.S.](#g-miscs)
+7. All titles of pull requests which add a new library must start with `[new]`.
+8. If you're the author/maintainer of a library, prepend `[author]` to the title when creating a pull request for that library. Such PRs will be processed with higher priority than others.
+9. Fill out all applicable fields in the PR description. This is essential to make review easier and faster.
+10. Once you have created a PR, our CI will run automated tests (generally takes few minutes). If the tests fail, take a look at the error(s) and fix them. You can always ask for help. Pull requests which fail our tests will not be merged.
+11. You might be asked to modify your commit(s) by one of our maintainers or the CI. Use `git commit --amend` or `git rebase` and force push the changes using `--force` to update your PR.
 
-## A. issue
+#### Update an Existing Library
 
-### a. general conventions
+1. Follow already existing file and directory structure. If you are fixing an incorrect structure, mention this in the commit description or pull request.
+2. Ensure that you update the `package.json` file to reflect the new version number.
+3. Ensure that `filename` in the `package.json` file is correct. File names might change with a new version.
+4. Add [auto-update config](/documents/autoupdate.md) for the library if it doesn't already exist.
 
-* Before opening a issue ticket, please check if there is/was already an issue on the same topic.
+#### Add a New Library With Assets
 
-* If you are going to open a issue about your library (means you are the author/maintainer or one of the main authors/maintainers), please add `[author]` on the issue topic, we'll give it a higher priority to process.
+* All libraries are stores in the `ajax/libs` directory. Each library has it's own directory in `ajax/libs` and each version of the library has it's own subdirectory. For example, jQuery v1.1.2 will be located at `ajax/libs/jquery/1.1.2/`.
+* [`package.json`](https://www.npmjs.org/doc/package.json.html) file
+    * The root of each library (for example, `ajax/libs/jquery`) must have a the `package.json` file. We store all library-related meta data in this file.
```

Comment:
"it's own directory .... it's own" ====>  change the it's into possessive  pronoun: "its own .... its own"

Thanks